### PR TITLE
Replaced deprecated package

### PR DESCRIPTION
**Pull request by a-a-GiTHuB-a-a**
`boom` is now deprecated. I replaced it with `@hapi/boom`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Add accepts parser to fastify
 
 ## Usage
 
-```js
+```nodejs
```
Comment (jsumners) [inline README.md:15]: This change is not necessary.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ See [accepts package](https://www.npmjs.com/package/accepts) for all available A
 
 This plugin adds to `Request` object all `Accepts` object methods.
 
-```js
+```nodejs
```
Comment (jsumners) [inline README.md:41]: This change is not necessary.

File: README.md (hunk 1 above)
Comment (Uzlopak) [reply README.md:15]:
```suggestion
```js
```

File: README.md (hunk 2 above)
Comment (Uzlopak) [reply README.md:41]:
```suggestion
```js
```

**Pull request merged**